### ISSUE TITLE
Install on linux fails with 1.43.3

### ISSUE THREAD

**Issue by jbockler** [Bearer/bearer#1590]
Our CI that does the bearer scan fails since the release of 1.43.3, because it can not install bearer.

## Description & Reproduction

We are using the install script for bearer:
`curl -sfL https://raw.githubusercontent.com/Bearer/bearer/main/contrib/install.sh | sh`

Platform: linux/amd64

## Expected Behavior

Bearer should be installed.

## Actual Behavior

Output:
```
Bearer/bearer info Checking dependencies...
git version 2.25.1
Bearer/bearer info checking GitHub for latest tag
Bearer/bearer info found version: 1.43.3 for v1.43.3/linux/amd64
```
And then the install script exists

## Possible Fix

I have seen that the Github release only contains the tarballs for darwin. Is this intentional?

## Your Environment

- Operating System and version: Ubuntu

**Comment by gotbadger** [comment]
@jbockler We had an issue yesterday where goreleaser somehow made a pre-release public. These pre releases do not build with all targets which would cause the install script to fail. We didnt notice this immediately and we have since released the full build (think about an hour after this issue was opened).

I've verifed this on a linux/amd64 ubuntu:18.04 docker image and it appears to be working now. Let me know if you are still having issues and I can investigate more.

FYI we are currently investigating what happened and guarding a bit more against this with changes in #1591
